wgpu-types/src/lib.rs

```diff
@@ -1462,6 +1464,8 @@ pub struct DeviceDescriptor<L> {
     /// Exactly the specified limits, and no better or worse,
     /// will be allowed in validation of API calls on the resulting device.
     pub required_limits: Limits,
+    /// Specifies if experimental features are allowed to be enabled.
```

Comment:
This is not clear if the reader doesn’t already know where experimental features are found. It could be interpreted in such incorrect ways as “if this is enabled, wgpu will use some experimental features somewhere”.

```suggestion
    /// Specifies whether `self.required_features` is allowed to contain experimental features.
```